internal/storage/config.go

```diff
@@ -41,7 +41,7 @@ func CurrentToken(cs ConfigStore, ts TokenStore) (Token, error) {
 		return Token{}, err
 	}
 
-	return ts.Get(cfg.CurrentToken, false)
+	return ts.Get(cfg.CurrentToken, true)
```

Comment:
Why this change?

Reply:
It's not used anywhere anymore. The only time we use --reveal-tokens is on list.